vuepress/docs/docs/getting-started/README.md

```diff
@@ -180,6 +186,33 @@ IP=$(hostname -I | awk '{print $1}')
 sed -i "s/192.168.64.25/$IP/" entando.yaml
 ```
 
+### Define resource limits
```

Comment:
Could we move this to a dedicated page, perhaps under docs/reference and then linked from here? I think this is great information but probably not needed on the getting started page which we're trying to keep as short as possible.

Reply:
@firegloves , I can try to fold this into my 6.2 work if that would be helpful but let me know what you think.